### ISSUE TITLE
How to use port 80 rather than random port? ex: test.codenvy.io:80

### ISSUE THREAD

**Issue by putuoka** [codenvy/codenvy#2655]
i tried to chose PHP container with apache as web server. Everytime i start apache it always comes with random port for example: server1.codenvy.io:3455 
can i use port 80 rather than random ip as my web address?

**Comment by ghost** [comment]
@putuoka no, port publishing happens on a random port from the ephemeral port range.

**Comment by marteiro** [comment]
I understand the random port policy. but I would like to know if is there any alternative to this. My team is searching for a online IDE, we tried a bunch this week and Codeenvy sure seems the best. But we would share a URL so our team can approve development around the world. And since in big corporations this range of port sometimes are closed this is the only downtime for us. 

Some online IDEs use random subdomain instead of random port. Any solutions to this? 

Thanks,
